extensions/interactions/customization-args-defs.ts

```diff
@@ -85,26 +85,26 @@ export interface IGraphInputCustomizationArgs {
   graph?: {
     value: IGraphBackendDict;
   };
-  canAddVertex?: {
-    value: string;
+  canAddVertex: {
+    value: boolean;
```

Comment:
How did the type change from  string to boolean? The name implies it is boolean, but what string was it before?

Reply:
Oh, that was a mistake actually. :sweat_smile:

Reply:
Do you mean the 'string' or the change to 'boolean'?

Reply:
'string'

Reply:
Ah ok, so it is actually supposed to be a boolean, right?

Reply:
yup

Reply:
Ok!